File: src/Business/Services/CorrespondentGroupService.java
package Business.Services;

import Business.Services.Base.*;
import Common.DataAccess.*;
import Common.Models.*;
import Common.Services.*;

public class CorrespondentGroupService extends ReferenceParentEntityService<CorrespondentGroup, Correspondent> implements ICorrespondentGroupService {
    public CorrespondentGroupService(
            IGlobalDataAccess globalDataAccess,
            ICorrespondentGroupDataAccess entityDataAccess,
            ICorrespondentDataAccess childEntityDataAccess) {
        super(globalDataAccess, entityDataAccess, childEntityDataAccess);
    }
}

File: backend/Business.Impl/Services/CorrespondentGroupService.cs
using Business.Contracts.Services;
using Business.Impl.Services.Base;
using Business.Models.Entities;
using DataAccess.Contracts;
using Shared.Contracts;

namespace Business.Impl.Services;

internal sealed class CorrespondentGroupService : GroupService<CorrespondentGroup, Correspondent>, ICorrespondentGroupService
{
	public CorrespondentGroupService(ISharedContext sharedContext, IAppUnitOfWork unitOfWork)
		: base(sharedContext, unitOfWork, unitOfWork.CorrespondentGroupRepo, unitOfWork.CorrespondentRepo)
	{
	}
}
